src/index.css

```diff
@@ -1,13 +1,26 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.arrow {
+  width: 8px;
+  height: 5px;
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: Source Sans Pro, sans-serif;
+}
+
+.movy {
```

Comment:
do not use global CSS. use css modules for component specific styling. same comment for the styles that are under this one